app/models/post.rb

```diff
@@ -1,6 +1,6 @@
 class Post < ApplicationRecord
-  belongs_to :user, foreign_key: 'author_id'
-  belongs_to :comments
+  belongs_to :author, class_name: 'User'
+  has_many :comments
```

Comment:
- [ ] You did a great job working on your relationships! 💯 However, please also include the `one-to-many` relationship between POST and LIKE here! A POST has many LIKES. 🤓 🔧 

Also, according to the requirements:

![image](https://github.com/AbbasSarwar/BlogApp/assets/77069011/109f3e25-b8f1-4562-a77a-39eb5d1a48c4)

- [ ] Please consider fixing your `post_counter` function, which currently is throwing this error:

![image](https://github.com/AbbasSarwar/BlogApp/assets/77069011/4b041496-fb5f-42db-9513-a67ccd6f58d8)

This happens because the `Post` model doesn't have a property named `posts` this property belongs to the author of the post. consider fixing it! 🤓 🔧 

- [ ] Please consider adding a function that returns the last 5 comments for the post! 🤓 🔧